Luminary099/UPDATE_PROGRAM.agc

```diff
@@ -149,17 +155,20 @@
 V70UPDAT	CAF	UP70		# COMES HERE ON V70E
 		TCF	V73UPDAT +1
 
+
 V71UPDAT	CAF	UP71		# COMES HERE ON V71E
 		TCF	V73UPDAT +1
 
+
 V72UPDAT	CAF	UP72		# COMES HERE ON V72E
 		TCF	V73UPDAT +1
 
+
 V73UPDAT	CAF	UP73		# COMES HERE ON V73E
 
  +1		TS	UPVERBSV	# SAVE UPVERB UNTIL IT'S OK TO ENTER P27
 
-		TC	TESTXACT	# GRAB DISPLAY IF AVAILABLE, OTHERWISE
+		TC	TESTXACT	# GRAB DISPLAY IF AVAILABLE. OTHERWISE
```

Comment:
```suggestion
		TC	TESTXACT	# GRAB DISPLAY IF AVAILABLE, OTHERWISE
```

This is a comma in the scans 
![image](https://github.com/user-attachments/assets/856b6172-4bb3-43d7-b113-e095c0c4bfc8)

Reply:
Ah, I was not able to tell if that was a comment or a period, adding that change also.